ansible/roles/nginx/vars/cas_ssl.yml

```diff
@@ -11,6 +11,11 @@ nginx_sites:
     - name: "{{ deploy_env }}_cas_formplayer"
       hosts: formplayer
       port: "{{ formplayer_port }}"
+   limit_req_zones:
+    - name: "server_name"
+      zone: "$server_name"
+      size: "10m"
+      rate: "50r/s"
```

Comment:
This rate and burst are arbitrary and should likely be changed, but wasn't sure what we were expecting per second